src/TestEmail.php

```diff
@@ -251,9 +251,7 @@ final public function assertHasFile(string $expectedFilename, string $expectedCo
             /** @var ParameterizedHeader $header */
             $header = $attachment->getPreparedHeaders()->get('content-type');
 
-            Assert::that($header->getBodyAsString())
-                ->is($expectedContentType.'; name='.$expectedFilename)
-            ;
+            Assert::that($header->getValue())->is($expectedContentType);
```

Comment:
I don't think there's a need to assert on the filename here because it was matched above.